### PR TITLE
restore-storybook

### PR DESCRIPTION
**Pull request by DarioR01**
**Overall change:**
Fixes Storybook and Chromatic.

**Code changes:**

- Updates storybook glob pattern to match stories in their new locations
- Fixes wrong paths in storybook directory

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/legacy/components/MediaPlayer/index.stories.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import { boolean, withKnobs } from '@storybook/addon-knobs';
 import { CanonicalMediaPlayer, AmpMediaPlayer } from '.';
-import ampDecorator from '../../../../.storybook/helpers/ampDecorator';
+import ampDecorator from '../../../../../.storybook/helpers/ampDecorator';
```
Comment (RetiredBlob) [inline src/app/legacy/components/MediaPlayer/index.stories.jsx:5]: I may be wrong, but can't we import relative to the root just like this?
```suggestion
import ampDecorator from '.storybook/helpers/ampDecorator';
```

Comment (RetiredBlob) [reply src/app/legacy/components/MediaPlayer/index.stories.jsx:5]: Could alternatively alias the root, to something like `#`

**Review by RetiredBlob — APPROVED** [review]
👍

File: src/app/legacy/components/MediaPlayer/index.stories.jsx (hunk 1 above)
Comment (jroebu14) [reply src/app/legacy/components/MediaPlayer/index.stories.jsx:5]: hmm I don't think absolute paths are supported without some extra configuration

**Review by amoore108 — APPROVED** [review]

File: src/app/legacy/components/MediaPlayer/index.stories.jsx (hunk 1 above)
Comment (jroebu14) [reply src/app/legacy/components/MediaPlayer/index.stories.jsx:5]: Sorry, I'm not keen on setting up a new alias for a legacy component. I actually think that aliases hide bad file organisation. A better fix would be not colocating the ampDecorator in the storybook. But again, I'm no so keen on investing any time on making legacy components better.

**Pull request merged**